test/ok_test.exs

```diff
@@ -71,7 +71,16 @@ defmodule OKTest do
     assert result == {:ok, 2}
   end
 
-  test "primitives as final operation" do
+  test "primitives as final operation - ok literal" do
```

Comment:
If a test is completely duplicated in the module docs it can be deleted from this file.